doc/api/buffer.md

```diff
@@ -175,8 +175,8 @@ string into a `Buffer` as decoding.
   encoding a `Buffer` to a string, this encoding will omit padding.
 
 * `'hex'`: Encode each byte as two hexadecimal characters. Data truncation
-  may occur when decoding strings that do exclusively contain valid hexadecimal
-  characters. See below for an example.
+  may occur when decoding strings that do not exclusively contain valid
+  hexadecimal characters. See below for an example.
```

Comment:
Unfortunately, it doesn't matter much if the strings `exclusively contain hexadecimal characters`. In fact, it might be more surprising that `Buffer.from` truncates data even when it only contains hex digits:

```js
// only hex, still truncated :(
Buffer.from('000', 'hex').toString('hex') === '00'
```

Reply:
Is it because it reads 2 characters at a time?

Reply:
Nvm, read the docs, which state:
```js
Buffer.from('1a7g', 'hex');
// Prints <Buffer 1a>, data truncated when data ends in single digit ('7').
```

So the last `'0'` is truncated because it's a single character, and the hex decoder expects pairs of characters.
So since the third `'0'` does not have a pairing character, it is truncated.

Perhaps the description should state that? You know, not only in the code example?